src/Klaviyo.php

```diff
@@ -1,69 +1,54 @@
 <?php
```

Comment:
This file won't be a part of the SDK. faulty push, need to delete this, my bad 😅